File: meu-teste/Models/Equipment_position_history.cs
namespace equipment_position_history.Model
{
    public class Equipment_position_history
    {
        public int id;
        public Guid Equipment_id { get; set; }
        public DateTime Date { get; set; }
        public Single Lat { get; set; }
        public Single Lon { get; set; }
    }
}

File: Aiko Teste/teste/src/main/java/TesteEstagioBackendV2/teste/Model/EquipmentPositionHistory.java
package TesteEstagioBackendV2.teste.Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Date;
import java.util.UUID;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class EquipmentPositionHistory {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;

    @ManyToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "equipment_id")
    private Equipment equipment;

    @Column
    private Date date;

    @Column
    private float lat;

    @Column
    private float lon;

}
